extra/datadog.sh

```diff
@@ -3,17 +3,16 @@
 # Setup Locations
 APT_DIR="$HOME/.apt"
 DD_DIR="$APT_DIR/opt/datadog-agent"
-DD_BIN_DIR="$DD_DIR/bin/agent"
+# Export DD_BIN_DIR to be used by the wrapper
+export DD_BIN_DIR="$DD_DIR/bin/agent"
 DD_LOG_DIR="$APT_DIR/var/log/datadog"
 DD_CONF_DIR="$APT_DIR/etc/datadog-agent"
-DATADOG_CONF="$DD_CONF_DIR/datadog.yaml"
+export DATADOG_CONF="$DD_CONF_DIR/datadog.yaml"
 
 # Update Env Vars with new paths for apt packages
 export PATH="$APT_DIR/usr/bin:$DD_BIN_DIR:$PATH"
-export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$APT_DIR/usr/lib/x86_64-linux-gnu:$APT_DIR/usr/lib"
-export LIBRARY_PATH="$LIBRARY_PATH:$APT_DIR/usr/lib/x86_64-linux-gnu:$APT_DIR/usr/lib"
```

Comment:
@jyee I removed all the export for paths related to static linking and compiling. I want to double check with you in case you had them here for a reason I am missing.

Reply:
@arapulido this should be fine, since we're adding them explicitly in the `bash -c` later